packages/slice-machine/lib/models/common/widgets/Image/components.tsx

```diff
@@ -86,45 +86,32 @@ export const ConstraintForm: React.FC<{
   if (!display) {
     return null;
   }
-  const requiredChar = required ? "*" : "";
-  const [field, _meta, helpers] = useField(prefix);
 
-  const createSetField =
-    (key: string, fn = (v: string): number | string => v) =>
-    (e: React.ChangeEvent<HTMLInputElement>) => {
-      helpers.setTouched(true);
-      const cast = fn(e.target.value);
-      const value =
-        e.target.type === "number"
-          ? typeof cast === "number" && isNaN(cast)
-            ? null
-            : cast
-          : cast;
-      helpers.setValue({ ...field.value, [key]: value });
-    };
+  const [_field, meta] = useField(prefix);
+
+  const requiredChar = required ? "*" : "";
+  const nameFieldName = `${prefix}.name`;
```

Comment:
OPT: if they're used only once I wouldn't do variables for that.

Reply:
done